lib/kennel.rb

```diff
@@ -54,56 +55,78 @@ class << self
   self.err = $stderr
 
   class Engine
-    attr_accessor :strict_imports
-
-    def initialize
-      @strict_imports = true
+    def initialize(
+      generate: true,
+      show_plan: false,
+      require_confirm: default_require_confirm?,
+      update_datadog: false,
```

Comment:
nit (or maybe for later) simpler keywords could be nice (but maybe also confusing)
```suggestion
      plan: false,
      update: false,
      require_confirm: default_require_confirm?,
```

Reply:
Yeah, confusing.

Does `plan` mean make a plan, or show a plan? Does `update` mean update `generated`, or update Datadog? For the sake of one extra word, I'd much rather clearer names.